openc3-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue

```diff
@@ -255,6 +255,8 @@ export default {
         'openc3-tool-tablemanager',
         'openc3-tool-tlmgrapher',
         'openc3-tool-tlmviewer',
+        'openc3-enterprise-tool-base',
+        'openc3-enterprise-tool-admin',
```

Comment:
I don't think these should be here

Reply:
The alternative is copy this whole file into enterprise and then add in the differences, or come up with some sort of included subfile.  This is the simplest for now and we can cleanup later.